### PR TITLE
blue color updated

### PR DESCRIPTION
**Pull request by gabrielo91**
# Description

Update the JS SDK Venmo buttons to a new shade of blue 

#008CFF

## Before:

image

## After:

image

[WIP] Since could there be more projects involved...

### CODE REVIEW

**Review by mstuart — APPROVED** [review]

**Comment by mstuart** [comment]
👍

File: src/ui/buttons/styles/color.js [hunk 1]
```diff
@@ -57,7 +57,7 @@ export const buttonColorStyle = `
     }
 
     .${ CLASS.BUTTON }[${ ATTRIBUTE.FUNDING_SOURCE }=${ FUNDING.VENMO }].${ CLASS.COLOR }-${ BUTTON_COLOR.BLUE } {
-        background: #009cde;
+        background: #008CFF;
```
Comment (gregjopa) [inline src/ui/buttons/styles/color.js:60]: @gabrielo91 I noticed this color code `#009cde` is used 7 times in color.js (https://github.com/paypal/paypal-checkout-components/search?q=009cde). Do we need to update all 7 or just this one occurrence?

Comment (gabrielo91) [reply src/ui/buttons/styles/color.js:60]: We had a discussion about that and yes, the value `#009cde` matches several times, however, the only one that affects the Venmo button is in that line. The other occurrences are part of the Paypal button and its variants, for example, in the box-shadow color

image

**Review by BriceShatzer — COMMENTED** [review]
I know the Venmo logo [we use](https://www.paypalobjects.com/paypal-ui/logos/svg/venmo-color.svg) along w/ their [published brand identity]https://venmo.com/about/brand/#colors) is to use `#3D95CE` as their primary blue, 

and I can't find any reference to "#008CFF" in the [UI docs](https://engineering.paypalcorp.com/confluence/display/PPUI/Color) or the various UI libraries ([vx](https://github.paypal.com/pages/PayPal-UI-R/pp-vx/#/v2-styles/colors), [react](https://github.paypal.com/pages/PayPal-UI-R/pp-react/?path=/story/pp-react-foundations-colors--default-story), or [figma](https://www.figma.com/file/LK59UcT6kRs08HvZ9QWUsO/PayPal-UI-2.0-Web?node-id=2174%3A0) )

Is this tied to a specific request from design and/or Venmo?

**Comment by gabrielo91** [comment]
@BriceShatzer it seems to be a new requirement, you can check [this Figma link](https://www.figma.com/file/1AeWCpGbAPq9nujQLVZddm/Brand-Guidelines?node-id=1%3A2) . Should we update something else?

File: src/ui/buttons/styles/color.js (hunk 1 above)
Comment (gabrielo91) [reply src/ui/buttons/styles/color.js:60]: So yes, venmo and paypal buttons share that color, however, according to @mstuart we should only update the one that affects to it

**Comment by gabrielo91** [comment]
Still wondering why coverage was affected 🤦🏻

File: src/ui/buttons/styles/color.js (hunk 1 above)
Comment (gregjopa) [reply src/ui/buttons/styles/color.js:60]: okay cool. Glad you hear you already considered this 👍

**Review by gregjopa — APPROVED** [review]
Looks good! I manually reviewed the screenshot diffs and I see the new Venmo blue being used:
Screen Shot 2021-05-24 at 1 08 25 PM

**Comment by gabrielo91** [comment]
Indeed, https://app.codecov.io/gh/paypal/paypal-checkout-components/compare/1641/changes don´t know why the coverage is failing, any ideas?

**Comment by gregjopa** [comment]
> Indeed, https://app.codecov.io/gh/paypal/paypal-checkout-components/compare/1641/changes don´t know why the coverage is failing, any ideas?

Can you try rebasing with master so your feature branch is up-to-date? That's my guess as to why it thinks coverage went down. It's missing some prerender tests that were recently added:

Screen Shot 2021-05-24 at 1 17 20 PM

**Review by elizabethmv — APPROVED** [review]

**Comment by mstuart** [comment]
Just got the green light to merge this

**Pull request merged**